tidb-cloud/_index.md

```diff
@@ -1,75 +1,100 @@
 ---
```

Comment:
Please confirm with FE whether we need to use a file name different from the existing `_index.md`. Generally, we don't allow files with the same file name in one repo.

Reply:
Files starting with _ can be duplicated.